### ISSUE TITLE
If you use multiple loadings, message does not change

### ISSUE THREAD

**Issue by morzius** [CarlosBonetti/jquery-loading#4]
I got 2 buttons with ajax on them, 

first one
...
               beforeSend: function () {
                        $('body').loading({
                            stoppable: false,
                            message: 'Unpublish...'
                        });
                    },
...

second one
...
               beforeSend: function () {
                        $('body').loading({
                            stoppable: false,
                            message: 'Publish...'
                        });
                    },
...

if i click "publish", then another button will have same message and vice versa

**Comment by CarlosBonetti** [comment]
Hi @morzius, thanks for the feedback!

That's a scenario I didn't consider when developing this plugin.

For now, the plugin uses the first call to create the plugin object and store it in the associated DOM (in your case, the body). The subsequent calls use the previously defined overlay and just start it.

For now, you can temporarily fix your problem doing:
```javascript
$('body').removeData('jquery-loading');
$('body').loading({
...
});
```
This will remove the previously defined overlay and start it all again.

I will be working on a permanent solution and posting the feedback here. Let me know if you have another problem.

**Comment by morzius** [comment]
Thank you,
I did something similar

```javascript
$('body').loading({ 
...
});
$(".loading-overlay-content").text('...');
```

**Comment by tompazourek** [comment]
I solved it using this:

```js
$.Loading.setDefaults({
  onStop: function (loading) {
    $(loading.element).removeData('jquery-loading');
  }
});
```

**Comment by CarlosBonetti** [comment]
@morzius, @tompazourek. I just fixed the problem on release 1.1.0.

Now subsequent calls to the plugin will override config options.

Please, let me know if you find any problems. Thanks for the feedback!
